downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,422 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.8.0 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201.8.0/
+active: 2201.8.0
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.8.0
+    - /downloads/swan-lake-release-notes/2201.8.0-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.8.0-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.8.0
+
+<em>2201.8.0 (Swan Lake Update 8) is the eighth update release of Ballerina Swan Lake, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2023R1 version of the Language Specification.</em> 
+
+## Update Ballerina
+
+Update your current Ballerina installation directly to 2201.8.0 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist update` to update to this latest distribution.
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+>**Info:** From this release onwards, you can verify Ballerina artifacts using the Cosign CLI and Rekor APIs. For more information, see [Verify Ballerina artifacts](/downloads/verify-ballerina-artifacts).
```

Comment:
We mentioned the same in update 7 too. So do we need this?

Reply:
Removed.